src/blkdrv/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+if(WITH_BLUESTORE)
+set(libblkdrv_srcs
```

Comment:
indent.

Reply:
also, might want to use `list(APPEND ..)` for the sake of consistency.

Reply:
Thanks for the suggestion. I'll update it.